python/mxnet/test_utils.py

```diff
@@ -1434,16 +1434,6 @@ def check_consistency(sym, ctx_list, scale=1.0, grad_req='write',
 
     return gt
 
-def list_gpus():
```

Comment:
not sure if removal is appropriate

Reply:
can you elaborate? there's no usages. It's a function of test utils. I would suggest to remove.

Reply:
searching for `"test_utils.list_gpus"` yields some 2000 results on google, which is a lot for something that has "no usages". deleting it would be breaking them.

Reply:
Many of them are our own files, but fine, I leave the old function in, up to you to remove in future releases.